spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -124,45 +101,50 @@ MicroProfile Health provides the ability to expose the health of an application
 to the underlying platform (ex: Kubernetes)
 
  - http://microprofile.io/project/eclipse/microprofile-health
- - https://github.com/eclipse/microprofile-health/releases/tag/4.0
+ - https://github.com/eclipse/microprofile-health/releases/tag/4.0.1
 
 [[mp-jwt-auth]]
-=== MicroProfile JWT Authentication 2.0
+=== MicroProfile JWT Authentication 2.0 (TODO 2.1?)
 
 MicroProfile JWT Authentication provides role based access control (RBAC) microservice endpoints using OpenID Connect (OIDC) and JSON Web Tokens (JWT).
 
  - http://microprofile.io/project/eclipse/microprofile-jwt-auth
  - https://github.com/eclipse/microprofile-jwt-auth/releases/tag/2.0
 
 [[mp-metrics]]
-=== MicroProfile Metrics 4.0
+=== MicroProfile Metrics 5.0
 
 MicroProfile Metrics provides a unified way for MicroProfile applications to export monitoring data to management agents.
 Metrics will also provide a common Java API for exposing their telemetry data.
 
  - http://microprofile.io/project/eclipse/microprofile-metrics
- - https://github.com/eclipse/microprofile-metrics/releases/tag/4.0
+ - https://github.com/eclipse/microprofile-metrics/releases/tag/5.0
 
 [[mp-open-api]]
-=== MicroProfile OpenAPI 3.0
+=== MicroProfile OpenAPI 3.1
 
 MicroProfile OpenAPI provides a unified Java API for the https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md[OpenAPI v3 specification] that all application developers can use to expose their API documentation.
 
  - http://microprofile.io/project/eclipse/microprofile-open-api
- - https://github.com/eclipse/microprofile-open-api/releases/tag/3.0
+ - https://github.com/eclipse/microprofile-open-api/releases/tag/3.1
+
+[[mp-telemetry]]
+=== MicroProfile Telemetry 1.0
+
+MicroProfile Telemetry
+Telemetry data is needed to power observability products.
 
-[[mp-opentracing]]
-=== MicroProfile OpenTracing 3.0
```

Comment:
This section should be removed. However, at the end of the section of MicroProfile Telemetry 1.0, we should state soemthing like : MicroProfile Telemetry 1.0 supercedes MicroProfile OpenTracing due to the merge of OpenTracing and OpenSensus into OpenTelemetry. As a result, MicroProfile 6.0 will not include MicroProfile OpenTracing, which will become a standalone specification. It is highly recommended to move away from MicroProfile OpenTracing and use MicroProfile Telemetry instead.